garage/tf/core/mlp.py

```diff
@@ -20,10 +20,12 @@ def mlp(input_var,
     It maps real-valued inputs to real-valued outputs.
 
     Args:
-        input_var: Input tf.Tensor to the MLP.
-        output_dim: Dimension of the network output.
-        hidden_sizes: Output dimension of dense layer(s).
-        name: variable scope of the mlp.
+        input_var (tf.Tensor): Input tf.Tensor to the MLP.
+        output_dim (int): Dimension of the network output.
+        hidden_sizes (list[int]): Output dimension of dense layer(s).
+            For example, (32, 32) means this MLP consists of two
+            hidden layers, each with 32 hidden units.
+        name (str): Network name, also the variable scope.
         hidden_nonlinearity: Activation function for
```

Comment:
what about these types?

Reply:
They are functions. Make it hidden_nonlinearity(function)?